Applications/WebEditor.qmd

```diff
@@ -54,7 +54,7 @@ Upon initial access to the provided link, users are presented with the following
 
     As previously mentioned, it is necessary to log in to interact with the server. A new window will appear when attempting to log in, as demonstrated below. While other options are sometimes available, the recommended authentication choice is the "Copernicus Data Space Ecosystem". For further information regarding various authentication methods or to seek assistance, you can always click on the "help" option at the top or contact us.
 
-    <centre><img src="_images/login.PNG"  width="80%" height="60%"></centre>
+    <centre><img src="_images/webeditor/editor_login.PNG"  width="80%" height="60%"></centre>
```

Comment:
note that this image is pretty outdated, it still contains "EGI" and "internal" options, which are dropped now

Reply:
updated the image